pmm-app/src/pmm-add-instance-app-panel/DiscoveryPanel/InstancesTable.tsx

```diff
@@ -63,17 +63,13 @@ const InstancesTable = props => {
             type = 'mysql';
             break;
         }
-        return (
-          <a onClick={() => onSelectInstance({ type: type, credentials: { ...Object.assign({}, element, credentials), isRDS: true } })}>
-            Start monitoring
-          </a>
-        );
+        return <a onClick={() => onSelectInstance({ type, credentials: { ...{ ...element, ...credentials }, isRDS: true } })}>Start monitoring</a>;
       },
     },
   ];
 
   return instances && instances.length ? (
-    <Table dataSource={instances} pagination={false} bordered={false} columns={columns} rowClassName={() => 'discovery-table-row'} size={'small'} />
+    <Table dataSource={instances} pagination={false} bordered={false} columns={columns} rowClassName={() => 'discovery-table-row'} size="small" />
```

Comment:
I suggest to change the line length to either 120 or 110 to avoid horizontal scrolling.